src/pages/contributors/vincent-latombe.adoc

```diff
@@ -0,0 +1,137 @@
+
+== What is your background prior to contributing to Jenkins?
+
+I graduated high school in 2001 and then, as is customary in France, attended preparatory school in order to go to engineering school.
+Only the students from the preparatory school with the best grades are selected to take the written and oral examinations for engineering school.
+Alternatively, some people decide to attend university, work towards being within the top of their class, and enter the engineering school that way.
+Unfortunately, I missed the top spots so I had to go for a third year.
+After finally achieving the marks I needed, I went to an engineering school in Grenoble and studied software engineering.
+For my final year, I studied in Sweden for eight months, then returned and completed an internship for my masters degree.
+After getting my degree, I was hired by Amadeus and worked for them for eight years.
+Initially, I worked on the product, but I quickly switched to the backend and tooling.
+
+In 2009 or 2010, during the early stages of CI implementation, the company had some Cruise Control instances deployed.
+Then we heard about a product called Hudson that was open source. 
+It had a nice web interface where you could edit your projects using the UI directly.
+Cruise control was driven from XML files as well, but every time you wanted to edit anything, you needed to restart the instance.
+It was nowhere as flexible as Hudson.
+When we started with Hudson, I started by picking one server, deploying a project with a branch, configuring limited availability, and seeing what we could do with it quickly.
+The SCM we were using at the time was Clear Case from IBM, which was designed to work within an IBM environment.
+However, there was a Clear Case plugin maintained by Andrew Bayer.
+When I started using Hudson, I began discovering some problems with my environment.
+
+I started to contribute fixes, and then after a while, we grew through a deployment.
+We started with building just the main branch, then we started considering more projects, finally leading up to building a development branch.
+My development with Jenkins grew organically over the years and that's how I got involved with the Jenkins project.
+Eventually, Andrew changed companies and left Clear Case, which meant that the Clear Case plugin was no longer being maintained.
```

Comment:
```suggestion
I started to contribute fixes, and then after a while, we grew through a deployment.
We started with building just the main branch of one project, then multiple projects, then all projects.
This was a game changer, especially in an environment where integration phases were a real challenge and people delivering features as quick as they could.
These would result in conflicts that were taking days, if not weeks to resolve, but getting a point of reference and tracking stabilization efforts were key.
Getting high performance hardware was also a challenge back then, since developers would take hours just to compile projects locally.
CPUs were 2 cores, magnetic disks, with no efficient way to parallelize.
My development with Jenkins grew organically over the years and that's how I got involved with the Jenkins project.
Eventually, Andrew changed companies and stopped maintaining ClearCase (to start using Git), leaving the ClearCase plugin up for adoption.
```